test/functionalTest/cases/1169_uri_param_options_map/uri_param_options_map.test

```diff
@@ -31,10 +31,10 @@ brokerStart CB
 
 #
 # 01. OK: Send a request with one valid option (count)
-# 02. OK: Send a request with two valid options (count + normalized)
+# 02. OK: Send a request with two valid options (count + keyValues)
 # 03. Error: Send a request with one invalid option (canonical)
-# 04. OK: Send a request with ALL valid options (count, normalized, values & text)
-# 05. Error: Send a request with ALL valid options and ONE invalid
+# 04. OK: Send a request with MANY valid options (count, keyValues, append)
```

Comment:
It is very hard to mantain a test with ALL options, given that ALL is changing very fast (at least in this early stages of NGSIv2 implementation... maybe it will be more stable in the future). Thus, I have opted to re-oriented the test using MANY.